File: ai_bypasser.py
import random
import re

# Simple synonym dictionary (subset for portability)
SYNONYMS = {
    'artificial': ['synthetic', 'man-made', 'computerized'],
    'intelligence': ['intellect', 'smartness', 'cognition'],
    'transforming': ['changing', 'revolutionizing', 'altering'],
    'world': ['globe', 'planet', 'society'],
    'automating': ['streamlining', 'mechanizing', 'simplifying'],
    'tasks': ['duties', 'jobs', 'activities'],
    'providing': ['offering', 'supplying', 'delivering'],
    'insights': ['perspectives', 'knowledge', 'ideas'],
    'widely': ['extensively', 'broadly', 'commonly'],
    'used': ['employed', 'utilized', 'applied'],
    'industries': ['sectors', 'fields', 'areas'],
    'healthcare': ['medicine', 'health', 'medical'],
    'finance': ['banking', 'economics', 'financial'],
    'education': ['schooling', 'learning', 'academia']
}

def get_synonym(word):
    """Get a random synonym for a word if available."""
    word = word.lower()
    return random.choice(SYNONYMS.get(word, [word])) if word in SYNONYMS else word

def vary_sentence(sentence):
    """Add basic sentence variation."""
    if random.random() < 0.3 and len(sentence.split()) > 5:
        # Split long sentences
        words = sentence.split()
        mid = len(words) // 2
        return ' '.join(words[:mid]) + '. ' + ' '.join(words[mid:]) + '.'
    if random.random() < 0.2:
        # Add discourse marker
        markers = ['Interestingly, ', 'In fact, ', 'Notably, ']
        return random.choice(markers) + sentence.lower()
    return sentence

def paraphrase_text(text):
    """Paraphrase text to reduce AI detection."""
    # Split into sentences
    sentences = re.split(r'(?<=[.!?])\s+', text.strip())
    paraphrased_sentences = []

    for sentence in sentences:
        words = sentence.split()
        new_words = []

        for word in words:
            # Preserve punctuation
            if word in '.,!?':
                new_words.append(word)
                continue
            # Replace with synonym (40% chance)
            if random.random() < 0.4:
                synonym = get_synonym(word)
                # Preserve capitalization
                if word[0].isupper():
                    synonym = synonym.capitalize()
                new_words.append(synonym)
            else:
                new_words.append(word)

        new_sentence = ' '.join(new_words)
        new_sentence = vary_sentence(new_sentence)
        paraphrased_sentences.append(new_sentence)

    # Join and clean up
    result = ' '.join(paraphrased_sentences)
    result = re.sub(r'\s+', ' ', result).replace(' ,', ',').replace(' .', '.')
    return result

def ai_bypass(text):
    """Main function to process text."""
    if not text.strip():
        return "Error: Input text is empty."
    return paraphrase_text(text)

# Example usage
if __name__ == "__main__":
    sample_text = "Artificial intelligence is transforming the world by automating tasks and providing insights."
    print("Original:", sample_text)
    print("Paraphrased:", ai_bypass(sample_text))

File: ai_bypasser.js
// Simple synonyms with basic, kid-friendly words
const SYNONYMS = {
    'artificial': ['fake', 'made-up', 'computer'],
    'intelligence': ['smartness', 'brain', 'thinking'],
    'transforming': ['changing', 'fixing', 'making new'],
    'world': ['earth', 'place', 'people'],
    'automating': ['doing', 'helping', 'making easy'],
    'tasks': ['jobs', 'work', 'things'],
    'providing': ['giving', 'showing', 'sharing'],
    'insights': ['ideas', 'thoughts', 'tips'],
    'widely': ['often', 'a lot', 'many'],
    'used': ['tried', 'picked', 'done'],
    'industries': ['work', 'jobs', 'places'],
    'healthcare': ['doctors', 'health', 'care'],
    'finance': ['money', 'banks', 'cash'],
    'education': ['school', 'learning', 'classes'],
    'big': ['large', 'huge', 'great'],
    'good': ['nice', 'great', 'cool'],
    'help': ['aid', 'support', 'boost'],
    'make': ['build', 'create', 'do']
};

function getSynonym(word) {
    // Get a random simple synonym or keep the word
    word = word.toLowerCase();
    const synonyms = SYNONYMS[word];
    return synonyms ? synonyms[Math.floor(Math.random() * synonyms.length)] : word;
}

function varySentence(sentence) {
    // Simple changes to make it less robotic
    const words = sentence.split(' ');
    if (Math.random() < 0.3 && words.length > 6) {
        // Split long sentences for kids
        const mid = Math.floor(words.length / 2);
        return words.slice(0, mid).join(' ') + '. ' + words.slice(mid).join(' ') + '.';
    }
    if (Math.random() < 0.2) {
        // Add a friendly starter
        const starters = ['Wow, ', 'Hey, ', 'Look, '];
        return starters[Math.floor(Math.random() * starters.length)] + sentence.toLowerCase();
    }
    return sentence;
}

function paraphraseText(text) {
    // Split text into sentences
    const sentences = text.match(/[^.!?]+[.!?]+/g) || [text];
    const paraphrased = sentences.map(sentence => {
        const words = sentence.trim().split(/\s+/);
        const newWords = words.map(word => {
            // Keep punctuation
            if (/[.,!?]/.test(word)) return word;
            // Swap with simple synonym (50% chance)
            if (Math.random() < 0.5) {
                const synonym = getSynonym(word);
                // Keep first letter case
                if (word[0] === word[0].toUpperCase()) {
                    return synonym.charAt(0).toUpperCase() + synonym.slice(1);
                }
                return synonym;
            }
            return word;
        });
        let newSentence = newWords.join(' ');
        newSentence = varySentence(newSentence);
        return newSentence;
    });
    // Clean up extra spaces and punctuation
    return paraphrased.join(' ').replace(/\s+/g, ' ').replace(/ ,/g, ',').replace(/ \./g, '.').trim();
}

function aiBypass(text) {
    // Main function to rewrite text
    if (!text.trim()) return "Error: Please add some text!";
    return paraphraseText(text);
}

// Example for testing
console.log("Original: Artificial intelligence is transforming the world.");
console.log("Paraphrased: " + aiBypass("Artificial intelligence is transforming the world."));
